datasets/ljspeech.py

```diff
@@ -0,0 +1,94 @@
+__all__ = ["DownloadLJSpeechCorpusJob", "LJSpeechCreateBlissCorpusJob"]
+
+import os
+import shutil
+import subprocess
+import wave
+
+from sisyphus import *
+
+from i6_core.lib import corpus
+from i6_core.util import uopen, check_file_sha256_checksum
+
+
+class DownloadLJSpeechCorpusJob(Job):
+    """
+    Downloads, checks and extracts the LJSpeech corpus.
+    """
+
+    def __init__(self):
+        self.out_ljspeech_folder = self.output_path("LJSpeech-1.1")
+        self.out_audio_folder = self.output_path("LJSpeech-1.1/wavs")
+        self.out_metadata = self.output_path("LJSpeech-1.1/metadata.csv")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        subprocess.check_call(
+            ["wget", "https://data.keithito.com/data/speech/LJSpeech-1.1.tar.bz2"]
```

Comment:
Do we have the option to download to a node-local temp location instead of the work folder?

Reply:
In the code right now not, but in principle I see no problem implementing it that way unless we want to be able to resume downloads and add the corresponding flags to wget.